src/config/config.ts

```diff
@@ -15,14 +15,21 @@ const rskRegtest = defineChain({
   },
 })
 
+const httpTransportConfig: HttpTransportConfig = {
+  batch: {
+    // this is the default value configured in RSKj
+    batchSize: 100,
+  },
+}
+
 export const config = createConfig({
   chains: [rskRegtest, rootstockTestnet, rootstock],
   transports: {
     [rootstock.id]: http(undefined, {
-      batch: true,
+      ...httpTransportConfig,
     }),
     [rootstockTestnet.id]: http(undefined, {
```

Comment:
```suggestion
    [rootstockTestnet.id]: http(undefined, httpTransportConfig),
```

Reply:
I don't want to use the same object, because I don't know if they mutate it internally.